pkg/awsutils/awsutils.go

```diff
@@ -1922,18 +1938,18 @@ func (cache *EC2InstanceMetadataCache) getENIsFromPaginatedDescribeNetworkInterf
 	return innerErr
 }
 
-// SetCNIUnmanagedENIs Set unmanaged ENI set
-func (cache *EC2InstanceMetadataCache) SetCNIUnmanagedENIs(eniID []string) error {
+// SetMultiCardENIs Set unmanaged ENI set
+func (cache *EC2InstanceMetadataCache) SetMultiCardENIs(eniID []string) error {
 	if len(eniID) != 0 {
-		cache.cniunmanagedENIs.Set(eniID)
+		cache.multiCardENIs.Set(eniID)
 	}
 	return nil
 }
 
-// IsCNIUnmanagedENI returns if the eni is unmanaged
-func (cache *EC2InstanceMetadataCache) IsCNIUnmanagedENI(eniID string) bool {
+// IsMultiCardENI returns if the eni is unmanaged
```

Comment:
nit: `IsMultiCardENI returns if the ENI is not behind the default network card index (multi-card ENI)`